src/coreclr/vm/jitinterface.cpp

```diff
@@ -2462,11 +2462,7 @@ static unsigned ComputeGCLayout(MethodTable* pMT, BYTE* gcPtrs)
 
 unsigned CEEInfo::getClassGClayout (CORINFO_CLASS_HANDLE clsHnd, BYTE* gcPtrs)
 {
-    CONTRACTL {
```

Comment:
All JIT/EE interface entrypoint use the expanded version of the contract for some reason. If you like the contract shorthand better, it would be nice to do a find&replace pass over the whole file to maintain consistent style (in separate PR).